src/core/thread/mesh_forwarder.hpp

```diff
@@ -63,6 +63,81 @@ enum
  * @{
  */
 
+/**
+ * This class reprents an IPv6 fragmentation priority entry
+ *
+ */
+OT_TOOL_PACKED_BEGIN
+class FragmentEntry
+{
+public:
+    /**
+     * This method returns the fragmentation datagram tag value.
+     *
+     * @returns The fragmentation datagram tag value.
+     *
+     */
+    uint16_t GetDatagramTag() { return mDatagramTag; }
+
+    /**
+     * This method sets the fragmentation datagram tag value.
+     *
+     * @param[in]  aDatagramTag  The fragmentation datagram tag value.
+     *
+     */
+    void SetDatagramTag(uint16_t aDatagramTag) { mDatagramTag = aDatagramTag; }
+
+    /**
+     * This method returns the fragmentation priority value.
+     *
+     * @returns The fragmentation priority value.
+     *
+     */
+    uint8_t GetPriority() { return mPriority; }
+
+    /**
+     * This method sets the fragmentation priotity value.
+     *
+     * @param[in]  aPriority  The fragmentation priority value.
+     *
+     */
+    void SetPriority(uint8_t aPriority) { mPriority = aPriority; }
+
+    /**
+     * This method returns the Fragment Priority entry's remaining lifetime.
```

Comment:
`Fragment Priority`, `fragmentation priority` and 'fragment priority' are mix-used. How about unifying them?